### ISSUE TITLE
comment

### ISSUE THREAD

**Issue by HarryLu95** [madelinezL/Project-Demos#1]
I would focus more on 1's recall in this project

**Comment by madelinezL** [comment]
Which project you were referring?

**Comment by HarryLu95** [comment]
> Which project you were referring?

The fraud detection one

**Comment by madelinezL** [comment]
Agreed, because we want to minimize the false negative - i.e. detect the potential fraud transactions as best as we could

**Comment by HarryLu95** [comment]
> Agreed, because we want to minimize the false negative - i.e. detect the potential fraud transactions as best as we could

Yes that's right! Like financial institutions always have fraud detection models, which aims to catch as much as real frauds and less false positives.
